src/core/opamStubsTypes.ml

```diff
@@ -114,7 +114,7 @@ type win32_version_info = {
     (** Non-fixed string table. First field is a pair of Language and Codepage ID. *)
 }
 
-external check_executable : string -> unit = "opam_check_executable"
+external is_executable : string -> bool = "opam_is_executable"
```

Comment:
`check_executable` was added in this PR, is it possible to have a single commit that adds it as `is_executable` ?